libs/sdk-ui-kit/src/AppearancePage/WhiteLabeling.tsx

```diff
@@ -0,0 +1,73 @@
+// (C) 2022 GoodData Corporation
+import React from "react";
+import { Button } from "../Button";
+import { TooltipIcon } from "./TooltipIcon";
+import { DocumentationLink } from "./DocumentationLink";
+import { LoadingSpinner } from "../LoadingSpinner";
+import { StylingComponent } from "./StylingComponent";
+import { StylingHeader } from "./StylingHeader";
+import { StylingFooter } from "./StylingFooter";
+
+const ALIGN_POINTS = [{ align: "cr tl" }, { align: "bc tc" }, { align: "bc tr" }];
+const ARROW_OFFSETS = { "cr tl": [16, -12], "bc tc": [0, 16], "bc tr": [12, 16] };
+
+/**
+ * @internal
+ */
+export interface IWhiteLabelingProps {
+    title: string;
+    status: string;
+    tooltip: string;
+    linkText: string;
+    linkValue: string;
+    actionText: string;
+    isLoading: boolean;
+    onSubmit: () => void;
+}
+
+/**
+ * @internal
+ */
+export const WhiteLabeling = (props: IWhiteLabelingProps) => {
```

Comment:
There is nothing in the component that tides it to white-labeling. Everything is generic. I would just name it `SettingWidget`, maybe `SimpleSettingWidget` as this is widget with limited ability to just toggle one setting and explain what is it about. If we introduce one more similar setting, it would be weird to either use the `WhiteLabeling` component for it or just duplicate it needlesly.

Reply:
Also, please use `React.FC` type.